### ISSUE TITLE
Sidekiq Admin UI Access Request for Kristen Brown

### ISSUE THREAD

**Issue by kristen-brown** [department-of-veterans-affairs/va.gov-team#56922]
#### Access to the Sidekiq Admin UI is managed through a GitHub Teams per environment via the following teams: 
- [Dev](https://github.com/orgs/department-of-veterans-affairs/teams/va-gov-sidekiq-dev)
- [Staging](https://github.com/orgs/department-of-veterans-affairs/teams/va-gov-sidekiq-staging)
- [Sandbox](https://github.com/orgs/department-of-veterans-affairs/teams/va-gov-sidekiq-sandbox)
- [Prod](https://github.com/orgs/department-of-veterans-affairs/teams/va-gov-sidekiq-prod)

_Please provide your GitHub username to be added to that team, along with a reason access is required and which environments you're requesting access to._

- GitHub username: @kristen-brown 
- Reason access is required: I am an engineer working on the Lighthouse Benefits & Appeals APIs and require access in order to review and troubleshoot Sidekiq jobs. I am requesting access to all environments because I am one of the engineers on call for production support for the Lighthouse Benefits Intake, Appeals, and Forms APIs.
- Environments requesting access to: All environments

**Comment by ryan-mcneil** [comment]
Access granted to `dev`, `staging`, and `sandbox` environments.

Access to `prod` is pending approval by OCTODE, and we'll update you as soon as it's available.

**Comment by ryan-mcneil** [comment]
You've been granted access to all environments, with access to Production limited to 6 months, expiring 11/5/23

**Comment by ryan-mcneil** [comment]
slack reminder [here](https://dsva.slack.com/archives/C02RLAUM2LV/p1683306391067539)

**Comment by kristen-brown** [comment]
@ryan-mcneil Thank you!

**Comment by bdech** [comment]
@kristen-brown Your Sidekiq Admin UI Production access has expired. Would you like to request an extension of access?

**Comment by kristen-brown** [comment]
@ryan-mcneil Yes, thank you! I still require access to the Sidekiq Admin UI in Production.

**Comment by bdech** [comment]
slack reminder [here](https://dsva.slack.com/archives/C02RLAUM2LV/p1699293378876629)

**Comment by kristen-brown** [comment]
Thank you! I'll set up reminder notification for next time.

**Comment by kristen-brown** [comment]
Hello! I see that my Sidekiq UI Production access is expiring on 5/6/24, and I do still require this access. Can my access be extended? Thank you!

CC: @bdech

**Comment by davidpetras** [comment]
@kristen-brown  your Production access has been extended for 6 months,
expiring 11/6/24.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by davidpetras** [comment]
Slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1714162453648299)

**Comment by kristen-brown** [comment]
Hello! I see that my Sidekiq UI Production access is expiring on 11/6/24, and I do still require this access. Can my access be extended? Thank you!

**Comment by kristen-brown** [comment]
Hello! I see that my Sidekiq UI Production access is expiring on 11/6/24, and I do still require this access. Can my access be extended? Thank you!

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a 👍 to extend their access?

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1730308391098019)

**Comment by kristen-brown** [comment]
Hello! I see that my Sidekiq UI Production access is expiring on 5/3/25, and I do still require this access. Can my access be extended? Thank you!

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
@kristen-brown, your Production access has been extended for 6 months, expiring 11/3/25.

Please set a reminder for yourself to return to this Issue and request an extension within 1 week of the expiration date. You'll
receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require you to fill out a new request.

**Comment by janetcc22** [comment]
Approved

**Comment by GWright321** [comment]
Slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1764019675105349)
